addons/stock_account/models/product.py

```diff
@@ -114,11 +109,6 @@ class ProductProduct(models.Model):
     stock_value = fields.Float(
         'Value', compute='_compute_stock_value')
 
-    @api.onchange('type')
-    def onchange_type_valuation(self):
-        # TO REMOVE IN MASTER
-        pass
-
     @api.multi
     def do_change_standard_price(self, new_price, account_id):
         """ Changes the Standard Price of Product and creates an account move accordingly."""
```

Comment:
Valuation is now changed on the category.  Indeed, adding the product_id on the account move would not be bad.  (if not already, saas-18)